pkg/release/release.go

```diff
@@ -188,7 +188,6 @@ func GetK8sRef() string {
 // GetK8sRef() point to their default values.
 func IsDefaultK8sUpstream() bool {
 	return GetK8sOrg() == DefaultK8sOrg &&
-		GetK8sRepo() == DefaultK8sRepo &&
```

Comment:
Hm, I'm wondering why we modify this :thinking:

Reply:
this funcition is used in the FF and now we will need expand to other repo and not only for k/k